### ISSUE TITLE
Updating a profile results in a new profile

### ISSUE THREAD

**Issue by prinyo** [Nexus-Mods/NMM-Test-Builds#5]
![image](https://cloud.githubusercontent.com/assets/18665141/14793004/4cbb04d4-0b1d-11e6-921e-b88139f58f46.png)

You can see on the right side there are 3 different profiles "small test" - they are all the same profile and were created by trying to update from the local one. On the site they are shown with the same name and added number,

**Comment by AraFox** [comment]
+1

**Comment by roamwild** [comment]
Works for me properly, now:
-select desired mod in Local pane
-click 'Backup' cloud button
-select YES to update this profile, 
       spammed until INVALID KEY did not return (6 or 7 times)
profile was updated to (v.2) on Nexus
- did this again and after spamming past INVALID KEY (3-5 times) again,
  profile was updated to (v.3)

https://www.nexusmods.com/fallout4/profiles/2195/

brilliant!

**Comment by DuskDweller** [comment]
This issue is being looked at, it'll be great if you could provide the exact steps to replicate it.

**Comment by AraFox** [comment]
you know I'm glad you asked, because the results I'm getting are making me think I'm doing it wrong, and if I am then please tell me because I will represent your average "I am not a programmer, coder, or any other kind of magician" user and if it isn't intuitive to me, it probably won't be to your average user. In that light, I've recorded myself failing to update my profile. I used a profile that was already present on the nexus profile library "FoxTest62.2_2" and I added a handful of mods and just tried to upload and share. The end result was two profiles named "FoxTest62.2_2" If there is another way to do this to make it overwrite the profile already in the library with the newer version, I'm just not seeing it - which is **_entirely_** possible. 

https://www.youtube.com/watch?v=LpfF1kfwlao  I'm not exactly a heavily followed you tuber :/ but will be happy to remove the video once you've got what you need if you'd like. Either way, please let me know if I'm missing the magic "update an already existing profile" button.

**Comment by prinyo** [comment]
![image](https://cloud.githubusercontent.com/assets/18665141/14855089/a17ca65a-0c92-11e6-9997-0f08e7aa170d.png)

The steps that I did are:
1. There was no local "small test_2" profile. I restored it from the online profile. 
2. Imediatelly after that I clicked the cloud with the arrow up icon and it created a new profile - now online there are 2 different "small test_2"
3. Now when I click the refresh icon I see that the name is small test_2_1

![image](https://cloud.githubusercontent.com/assets/18665141/14855220/2d8f3f5e-0c93-11e6-9d95-40e847433b25.png)

Clicking the icon to backup without sharing produces the same result. Now I have "small test_2_2"
